pkg/protocol/engine/attestation/attestation.go

```diff
@@ -0,0 +1,20 @@
+package attestation
+
+import (
+	"github.com/iotaledger/hive.go/ads"
+	"github.com/iotaledger/hive.go/runtime/module"
+	"github.com/iotaledger/iota-core/pkg/protocol/engine/blocks"
+	iotago "github.com/iotaledger/iota.go/v4"
+)
+
+type Attestation interface {
```

Comment:
Should we rename this inferface? It is confusing with an actual iotago.Attestation

Reply:
`Attestations` (plural) maybe?